scripts/zkai_attention_derived_d128_outer_proof_route_gate.py

```
@@ -0,0 +1,665 @@
+    for spec in CANDIDATE_SPECS:
+        path = ROOT / spec["path"]
+        exists = path.exists()
+        entry = dict(spec)
+        entry["exists"] = exists
+        if exists and path.is_file():
+            raw = COMPRESSION.read_source_bytes(path, f"candidate {spec['name']}")
+            entry["file_sha256"] = sha256_bytes(raw)
+        else:
+            entry["file_sha256"] = None
+        if exists != spec["expected_exists"]:
+            raise AttentionDerivedD128OuterProofRouteError(f"candidate existence drift: {spec['name']}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)